client/src/components/Common/selectCurrency/currencies.js

```diff
@@ -0,0 +1,29 @@
+const flagOptions = [
+  {
+    value: 'USD',
+    text: 'دولار امريـكي',
```

Comment:
you can change the Arabic names like 
[this site:](https://www.xe.com/ar/currencyconverter/)